tgarchive/example/template.html

```diff
@@ -39,163 +40,171 @@
 			<header class="header">
 				<div class="logo">
 					<a href="{{ config.site_url }}"><img src="static/logo.svg" alt="" /></a>
-					<p class="desc">
+					<div class="desc">
```

Comment:
div is more flexible than p, as div allows nesting

Reply:
In this context, it is always going to be an auto generated line of text. `<p>` is thus fine. If someone wants to add multiple paragraphs there by editing the template, they can always change the markup.